client/me/help/help-contact-form/index.jsx

```diff
@@ -269,6 +307,44 @@ export class HelpContactForm extends React.PureComponent {
 		);
 	};
 
+	/**
+	 * For the forums: check if we're dealing with a WP.com site.
+	 */
+	analyseSiteData = () => {
+		const requestData =
+			this.state.userDeclaredUrl && ( this.state.siteData || this.state.errorData );
```

Comment:
Here `requestData` can be either the value of `this.state.siteData` or `this.state.errorData`. But both are very different data structures that have nothing in common. Or do they?

Reply:
I guess the only thing that they have common is they're both used to figure out if the site is WP.com-hosted or not. I've removed `requestData` now. :)